nucliadb_utils/nucliadb_utils/settings.py

```diff
@@ -25,7 +25,7 @@
 
 
 class RunningSettings(BaseSettings):
-    debug: bool = True
+    debug: bool = False
```

Comment:
This was the actual culprit. We're using this value to set the `debug` attribute of our fastapi apps, which, if set to True, makes the exception handlers be ignored and the full error traceback is returned